### ISSUE TITLE
Rabbit queue username/password

### ISSUE THREAD

**Issue by hazmat345** [beer-garden/beer-garden#62]
We need to be generating a (username?) password for each queue.

**Comment by loganasherjones** [comment]
Yes. Unique username and password. Even a unique queue name would be good so that it’s harder to intercept messages

**Comment by hazmat345** [comment]
We need to decide how to handle reconnects...

Are we going to have plugins tell us the username / password to use? What happens if they don't?

If we assign a username / password for a new queue, great. But what happens when that plugin disconnects and tries to reconnect? Should we be forcing remote plugins to persist their username / passwords?

**Comment by loganasherjones** [comment]
@hazmat345 are you handling this as part of #35 ?

**Comment by scott-taubman** [comment]
This is unlikely to be tackled in the current iteration of BG.  Closing for now.
